aws-attach-roles/iam-kubecost-savings-policy.json

```diff
@@ -0,0 +1,67 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Action": [
+                        "s3:GetBucketLocation",
+                        "s3:ListBucket"
+                    ],
+                    "Resource": [
+                        "arn:aws:s3:::YourThanosBucketName"
+                    ],
+                    "Effect": "Allow",
+                    "Sid": "kubecostMetricsBucket"
+                },
+                {
+                    "Action": [
+                        "s3:GetObject",
+                        "s3:ListBucket",
+                        "s3:ListBucketMultipartUploads",
+                        "s3:ListMultipartUploadParts",
+                        "s3:AbortMultipartUpload",
+                        "s3:PutObject"
```

Comment:
@jessegoodier Just curious, how do we come up with these IAM permissions? Just to make sure that it would not cause any issue for Thanos to interact with S3 bucket.

Reply:
good question. 
We should validate because I don't know who the original author was and I believe we can narrow it down.